index.html

```diff
@@ -44,5 +44,128 @@ <h2>I’am a Software Developer</h2>
+          <p class="Feat-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-stack-2">
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <p class="btn">See Project</p>
+        </div>
+        <div class="projects">
+          <h3>Profesional Art Printing Data</h3>
+          <p class="Feat-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-stack-2">
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <p class="btn">See Project</p>
+        </div>
+        <div class="projects">
+          <h3>Profesional Art Printing Data</h3>
+          <p class="Feat-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-stack-2">
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <p class="btn">See Project</p>
+        </div>
+      </div>
+    </main>
+
+    <section class="about-section">
+      <h2>About Me</h2>
+      <p class="Feat-text">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <sub-section class="btn">Get my Resume</sub-section>
+    </section>
+
+    <section class="skills">
+      <div class="skills-macro">
+        <div class="key-skills">
+          <img src="src/Rectangle-55.svg" alt="diamond icon" />
+          <h3>Language</h3>
+        </div>
+        <div class="skills-feat">
+          <p>Javascript</p>
+          <p>Ruby</p>
+          <p>Html</p>
+          <p>Css</p>
+        </div>
+      </div>
+
+      <div class="skills-macro">
+        <div class="key-skills">
+          <img src="src/Rectangle-57.svg" alt="square icon" />
+          <h3>Frameworks</h3>
+        </div>
+        <div class="skills-feat">
+          <p>Bootstrap</p>
+          <p>Ruby on Rails</p>
+          <p>RSPec</p>
+          <p>CapyBara</p>
+          <p>Selenium</p>
+        </div>
+      </div>
+
+      <div class="skills-macro">
+        <div class="key-skills">
+          <img src="src/Rectangle-56.svg" alt="round icon" />
+          <h3>Skills</h3>
+        </div>
+        <div class="skills-feat">
+          <p>Codekit</p>
+          <p>Github</p>
+          <p>Codepen</p>
+          <p>Gitlab</p>
+          <p>Terminal</p>
+        </div>
```

Comment:
- [ ] Please ensure that the technologies in each category (e.g., "JavaScript", "HTML" and "CSS") are also a list `ul` and each of them a `li` as per project requirements.